detectable/src/main/java/com/synopsys/integration/detectable/detectables/gradle/inspection/parse/GradleReportParser.java

```diff
@@ -39,26 +39,28 @@
 
     private final GradleReportConfigurationParser gradleReportConfigurationParser = new GradleReportConfigurationParser();
 
-    public Optional<GradleReport> parseReport(final File reportFile) {
+    public Optional<GradleReport> parseReport(File reportFile) {
         GradleReport gradleReport = new GradleReport();
         boolean processingMetaData = false;
-        final List<String> configurationLines = new ArrayList<>();
-        try (final InputStream dependenciesInputStream = new FileInputStream(reportFile); final BufferedReader reader = new BufferedReader(new InputStreamReader(dependenciesInputStream, StandardCharsets.UTF_8))) {
+        List<String> configurationLines = new ArrayList<>();
+        try (InputStream dependenciesInputStream = new FileInputStream(reportFile); BufferedReader reader = new BufferedReader(new InputStreamReader(dependenciesInputStream, StandardCharsets.UTF_8))) {
             while (reader.ready()) {
-                final String line = reader.readLine();
+                String line = reader.readLine();
                 /*
                   The meta data section will be at the end of the file after all of the "gradle dependencies" output
                  */
+                boolean shouldContinue = false;
                 if (line.startsWith(DETECT_META_DATA_HEADER)) {
                     processingMetaData = true;
-                    continue;
-                }
-                if (line.startsWith(DETECT_META_DATA_FOOTER)) {
+                    shouldContinue = true;
+                } else if (line.startsWith(DETECT_META_DATA_FOOTER)) {
                     processingMetaData = false;
-                    continue;
-                }
-                if (processingMetaData) {
+                    shouldContinue = true;
+                } else if (processingMetaData) {
                     setGradleReportInfo(gradleReport, line);
+                    shouldContinue = true;
+                }
+                if (shouldContinue) {
```

Comment:
If you switched to else if blocks, I am pretty sure you don't need to do this continue business.

Reply:
Using if-else blocks required me to use the `continue` statement twice which is against our Sonar policy.
I am very for removing/increasing that limitation.

Reply:
But I don't think you need the continues at all. Weren't they only there to do the work of the if else. Can't you just delete them?